### PR TITLE
Update tests for issue 1229

### PR DESCRIPTION
**Pull request by MSCetin37**
## Description

Update the test function as a results of this issue [942](https://github.com/opea-project/GenAIComps/pull/942)

## Issues

[1229](https://github.com/opea-project/GenAIExamples/issues/1229)

## Tests
GenAIExamples/DocSum/tests/test_compose_on_gaudi.sh
GenAIExamples/DocSum/tests/test_compose_on_xeon.sh

### CODE REVIEW

File: DocSum/tests/test_compose_on_xeon.sh [hunk 1]
```diff
@@ -150,23 +150,23 @@ function validate_microservices() {
...
+        "SUQzBAAAAAAAI1RTU0UAAAAPAAADTGF2ZjU4LjI5LjEwMAAAAAAAAAAAAAAA//tQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAASW5mbwAAAA8AAAAIAAAN3wAtLS0tLS0tLS0tLS1LS0tLS0tLS0tLS0tpaWlpaWlpaWlpaWlph4eHh4eHh4eHh4eHpaWlpaWlpaWlpaWlpcPDw8PDw8PDw8PDw+Hh4eHh4eHh4eHh4eH///////////////8AAAAATGF2YzU4LjU0AAAAAAAAAAAAAAAAJAYwAAAAAAAADd95t4qPAAAAAAAAAAAAAAAAAAAAAP/7kGQAAAMhClSVMEACMOAabaCMAREA" \
         "dataprep-video2audio" \
         "dataprep-video2audio-service" \
         "{\"byte_str\": \"$(input_data_for_test "video")\"}"
 
     # Docsum Data service - video
     validate_services \
         "${host_ip}:7079/v1/multimedia2text" \
-        '"query":"well"' \
+        '"query":"well' \
```
Comment (ashahba) [inline DocSum/tests/test_compose_on_xeon.sh:161]: There's a dangling `"` so needs to be reverted.

File: DocSum/tests/test_compose_on_xeon.sh [hunk 2]
```diff
@@ -150,23 +150,23 @@ function validate_microservices() {
...
         "dataprep-multimedia2text-service" \
         "dataprep-multimedia2text" \
         "{\"video\": \"$(input_data_for_test "video")\"}"
 
     # Docsum Data service - audio
     validate_services \
         "${host_ip}:7079/v1/multimedia2text" \
-        '"query":"well"' \
+        '"query":"well' \
```
Comment (ashahba) [inline DocSum/tests/test_compose_on_xeon.sh:169]: There's a dangling `"` so needs to be reverted.

**Review by ashahba — CHANGES_REQUESTED** [review]

**Comment by chensuyue** [comment]
Please also check this, https://github.com/opea-project/GenAIExamples/blob/eb87a70b6fd38d369ce062d618e45579c4b6dfaf/DocSum/tests/test_compose_on_xeon.sh#L153

File: DocSum/tests/test_compose_on_xeon.sh (hunk 1 above)
Comment (MSCetin37) [reply DocSum/tests/test_compose_on_xeon.sh]: fixed

**Comment by MSCetin37** [comment]
@chensuyue, @ashahba , @feng-intel, 
The original issue [1229](https://github.com/opea-project/GenAIExamples/issues/1229) is solved, but merge [992](https://github.com/opea-project/GenAIExamples/pull/992) is causing a DocSum Megaservice failure, which is not within the scope of this issue.

**Review by ashahba — APPROVED** [review]

**Review by lvliang-intel — APPROVED** [review]

**Pull request merged**